### PR TITLE
Fix/tr 83/prevent the next button to be clicked twice

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/TR-83

Make sure the `Next` button cannot be clicked twice in a row.

### CODE REVIEW

File: src/plugins/navigation/next.js [hunk 1]
```diff
@@ -208,36 +226,30 @@ export default pluginFactory({
...
+        };
 
         //create the button (detached)
         this.$element = createElement(isLastItem());
 
         //attach behavior
-        this.$element.on('click', function(e) {
+        this.$element.on('click', e => {
             e.preventDefault();
+            disableElement(this.$element);
```
Comment (wazelin) [inline src/plugins/navigation/next.js:243]: All right, so, this is basically the fix then.

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by habemuscode — APPROVED** [review]
Good refactor! LGTM

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Review by bziondik — APPROVED** [review]
Code review only
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**